docs/check-documentation/images.md

```diff
@@ -6,11 +6,14 @@ key | default | description
 ----|---------|------------
 `images.tagNotSpecified` | `error` | Fails when an image tag is either not specified or `latest`.
 `images.pullPolicyNotAlways` | `ignore` | Fails when an image pull policy is not `always`.
+`images.whitelist` | `error` | If the container image does not match the pattern in the whitelist, it will fail with an error.
```

Comment:
Thanks for the work here and docs! I think I am a little confused since it looks like the code supports both warning or error for lists of black/white lists. I think the docs could reflect that by maybe adding "`error` or `warning` (configurable)" to this table. Open to suggestions though!

Reply:
Fixed in the latest commit.